x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.test.tsx

```diff
@@ -291,23 +291,23 @@ describe('edit package policy page', () => {
       expect(renderResult.getByDisplayValue('Nginx description')).toBeInTheDocument();
     });
 
-    await act(async () => {
-      fireEvent.click(renderResult.getByText('Change defaults'));
-    });
+    fireEvent.click(renderResult.getByText('Change defaults'));
 
-    await act(async () => {
-      fireEvent.change(renderResult.getByDisplayValue('/var/log/nginx/access.log*'), {
-        target: { value: '' },
-      });
+    await waitFor(() => Promise.resolve(null));
```

Comment:
Hi @juliaElastic  The `waitFor(() => Promise.resolve(null))` has been added to account for instances where an async operation is performed so we can resolve whatever action might have been queued before proceeding the next part of the test. I've noticed that in this particular test suite, the test can also pass without including this, there's other parts of this test that absolutely require this, that been said it's really your call since you have more knowledge about this part of the code if we should keep it or not.

Reply:
We don't use this anywhere else in Fleet tests, so I don't think it's needed.

Reply:
Hi @juliaElastic I removed the usages in this file, however there were some places that still required it.